.github/workflows/main.yaml

```diff
@@ -109,7 +111,38 @@ jobs:
           name: otp_prebuilt_win32
           path: otp/release/win32/otp*.exe
 
+  build_macos:
+    name: Build Erlang/OTP on macOS
+    runs-on: macos-11
+    needs: pack
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install wxWidgets
+        run: brew install wxmac
+
+      - name: Download source archive
+        uses: actions/download-artifact@v2
+        with:
+          name: otp_git_archive
+
+      - name: Compile Erlang
+        run: .github/scripts/build-macos.sh
+
+      - name: Test Erlang
+        run: |
+          cd otp/otp
+          ./Install `pwd`
+          ./bin/erl -noshell -noshell -eval 'io:format("~s", [erlang:system_info(system_version)]), halt()'
+
+      - name: Upload tarball
+        uses: actions/upload-artifact@v2
+        with:
+          name: otp_prebuilt_macos
+          path: otp/otp_macos_*.tar.gz
+
   build:
+    # TODO: remove
+    if: github.repository == 'erlang/otp'
```

Comment:
Does this condition is needed here and in line 43?

Reply:
I should have clarified the TODO. This is a leftover from my fork where I didn't want to unnecessarily build things I didn't need. This and the other line will be removed before this lands.